### PR TITLE
A simple BMI calculator v1

### PR DESCRIPTION
**Pull request by MariusCD**
Enjoy with the app!

### CODE REVIEW

**Comment by diitaz93** [comment]
Nice feature @MariusCD !

File: mariuscd.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+height = float(input("Enter your height in cm: "))
```
Comment (diitaz93) [inline mariuscd.py:1]: Could you add as a header the webpage where you found the code?

**Review by diitaz93 — CHANGES_REQUESTED** [review]

File: mariuscd.py (hunk 1 above)
Comment (diitaz93) [reply mariuscd.py:1]: Commented with ###

Comment (MariusCD) [reply mariuscd.py:1]: Sure!

**Pull request merged**